FoodFriends/app/src/main/java/com/example/foodfriends/fragments/ProfileFragment.java

```diff
@@ -204,8 +209,9 @@ public void onClick(View v) {
             case R.id.ivAddPfp:
                 addPfp();
                 break;
-            case R.id.btnLogOut:
-                user_log_out();
+            case R.id.ivSettingsIcon://R.id.btnLogOut:
```

Comment:
removed by mistake?

Reply:
On purpose! Moved this logic to the settings page.